test/filename-case.js

```diff
@@ -184,6 +196,26 @@ ruleTester.run('filename-case', rule, {
 			'src/foo/___FOO-BAR.js',
 			'pascalCase',
 			'Filename is not in pascal case. Rename it to `___FooBar.js`.'
+		),
+		testManyCases(
+			'src/foo/foo_bar.js',
+			undefined,
+			'Filename is not in kebab case. Rename it to `foo-bar.js`.'
+		),
+		testManyCases(
+			'src/foo/foo-bar.js',
+			{camelCase: true, pascalCase: true},
+			'Filename is not in camel case or pascal case. Rename it to one of [`fooBar.js`, `FooBar.js`].'
```

Comment:
Doesn't look like `englishishJoinWords` is used here on the message.

Reply:
resolved